content/riak/kv/2.1.4/setup/upgrading/cluster.md

```diff
@@ -103,61 +99,61 @@ riak-admin transfers
 * While the node was offline, other nodes may have accepted writes on its
 behalf. This data is transferred to the node when it becomes available.
 
-8\. Repeat the process for the remaining nodes in the cluster
+8\. Repeat the process for the remaining nodes in the cluster.
 
->**Note for secondary index (2i) users**
->
->If you use Riak KV's [secondary indexes][usage secondary-indexes] and are
+{{% note title="Note for secondary index (2i) users" %}}
+If you use Riak KV's [secondary indexes][usage secondary-indexes] and are
```

Comment:
I don't know that this note is useful anymore. We don't support 1.3.1 or lower (let alone 1.4.z). Let's double check with @angrycub , but I think we can remove this.